scripts/cicd/verify-multiaccount-records-have-been-ingested.sh

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env bash
```

Comment:
Just /bin/bash is fine, env is appropriate for python / node / etc that generally are custom-installed